conf/views/aftSummary.njk

```diff
@@ -24,24 +26,109 @@
         }) }}
       {% endif %}
 
+      {% if formSearchText.errors.length > 0 %}
+        {{ govukErrorSummary({
+          "titleText": messages("error.summary.title"),
+          "errorList": formSearchText.errors
+        }) }}
+      {% endif %}
+
       <h1 class="govuk-heading-xl">
         <span class="govuk-caption-xl">{{ viewModel.schemeName }}</span>
         {{ messages('aft.summary.heading', quarterStartDate, quarterEndDate) }}
       </h1>
 
+       <form action="{{searchURL}}" method="post" autocomplete="off">
```

Comment:
can we not create a seperate component for search box as this page seems quite busy